script.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+
+let button = document.querySelector('.redBtn');
+let body = document.querySelector('.background');
+let container = document.querySelector('.vertical-center');
+let timeInfo = document.querySelector('.time-info');
+let lastUpdate = document.querySelector('.last-update');
+
+button.addEventListener('click', (event)=>{
+    pageChange([button.textContent, Date.now()]);
```

Comment:
А навіщо ускладнювати та передавати необхідні аргументи в масиві?
```suggestion
    pageChange(button.textContent, Date.now());
```